src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,68 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Astro DB can connect to any LibSQL server from any platform that exposes the Hrana protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a LibSQL database, set the following environment variables:
```

Comment:
```suggestion
To connect Astro DB to a libSQL database, set the following environment variables:
```

We tend to refer to `libSQL` with a lowercase `l`. I hope this isn't too much trouble to find and replace all 🫶